packages/toolkit/src/view/model/ModelHubCreatePageMainView.tsx

```diff
@@ -22,6 +23,7 @@ export const ModelHubCreatePageMainView = (
         breadcrumbs={["Models", "Model Settings"]}
         className="mb-10"
       />
+      <CreateModelForm1 accessToken={accessToken} enabledQuery={enableQuery} />
```

Comment:
@orangecoloured I think this a dev changes, I think we need to change it back

Reply:
I removed it.